stable/traefik/values.yaml

```diff
@@ -372,6 +372,15 @@ metrics:
     ## it to the dashboard service only
     restrictAccess: false
     # buckets: [0.1,0.3,1.2,5]
+  serviceMonitor:
```

Comment:
Could you have the service monitor key commented out?
I know that it would require an extra test in the template, but it would allow for the best backwards compatibility.

```
#serviceMonitor:
#  enabled: false
```